### DISCUSSION TITLE
Not able to run test_scene

### DISCUSSION THREAD

**Discussion by ahnupsingh** [empriselab/RCareWorld#160] [Help - Setting up / Configuring] [answered]
**Command**
```python test_scene.py```

```
Waiting for UnityEditor play...
Waiting for connections on port: 5004...
```

What are the prerequisites for running this ? Should i have to spin up some server on 5004 ?

**Comment by YoruCathy** [comment, marked as answer]
Hi, make sure you are on the latest commit of phy-robo-care. Also, make sure you did git lfs pull to download the large files.
